### ISSUE TITLE
cookie banner - position on the page

### ISSUE THREAD

**Issue by jolanta1** [tollwerk/TYPO3-ext-tw_eprivacy#19]
The cookie dialog should be the first element in the page's <body>, not the last, so that screen reader users will notice it first.
